README.md

```diff
@@ -11,6 +11,11 @@ This document outlines the setup and usage of ZenHub's API: a RESTful API with J
    repository](#get-the-zenhub-board-data-for-a-repository)
   - [Get Epics for a repository](#get-epics-for-a-repository)
   - [Get Epic data](#get-epic-data)
+  - [Move issue between pipelines](#move-issue-between-pipelines)
```

Comment:
The use of upper case and lower case should be consistent, also some articles are missing.